### DISCUSSION TITLE
the status alert

### DISCUSSION THREAD

**Discussion by Cheney-77** [jertel/elastalert2#1266] [Q&A] [answered]
if the operating system(like Linux or Windows server) is down,how can i use elastalert to send alert message?

**Comment by jertel** [comment, marked as answer]
You will redirect syslog (Linux) or event logs (windows) from the OS to Elasticsearch, possibly via a Logstash pipeline. Once the logs are in Elasticsearch you can use ElastAlert 2 to detect anomalies for that system, or use a flatline rule to detect a lack of new messages, which would indicate an outage. Based on the broad question you have asked I think you are going to need to read up on Elasticsearch and ElastAlert 2 documentation.

**Reply by Cheney-77** [reply]
got it,tkx
